Android/build.gradle

```diff
@@ -1,12 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.7.10"
+    ext {
+        kotlin_version = "1.6.10"
+        compose_ui_version = '1.2.1'
+    }
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:7.3.1'
```

Comment:
Changing the gradle version to `7.3.1` will cause `liboboe.so` not found, at least on my dev environment build. See [my issue](https://github.com/google/oboe/issues/1609).
Not sure if this is true on yours.

Reply:
Thanks for the name :)
For USB, I think asynchronous is the protocol we need. On Android side, it should be easy thanks to Class that already exist. 
On Windows side, I saw libusb which is a cross-platorm C library. 
But I never written in C# and I don't even know how to build the windows side project (I tried visual studio but I was lost when we have to choose what library we want).

I don't remember why I changed the gradle version, I think that was proposed to me but yes, it works fine to me. 
I tried to roll back to `4.2.2` but I've got an error (maybe because I used recent dependency idk).

Reply:
Sounds good, that is probably issue of my local Android Studio. I'll try to reinstall it and see if it gets fixed.

Reply:
Regarding `libusb`, I just took a look, and it seems to require Android device to have USB host (& OTG) support (see [wiki](https://github.com/libusb/libusb/wiki/Android#does-libusb-support-android)). Host support is documented [here](https://developer.android.com/guide/topics/connectivity/usb/host).

According to [stackoverflow](https://stackoverflow.com/a/14099963/11397618), it seems that host support exists only if the Android device has a USB port (which is not true on most phones).\
So another option is to let Android phone act in [USB accessory](https://developer.android.com/guide/topics/connectivity/usb/accessory) mode, and write [AOA](https://source.android.com/docs/core/interaction/accessories/protocol) code to talk to Android from Windows side (in C#).

But anyway, please let me know if you're able to setup the USB connection🙂

Reply:
I tried to implement the prop a while ago on Linux but it only allows bulk transfer. I couldn't test the audio but I managed to transmit data. I'll test with Windows in C# later I think, see what's better. Anyway, thanks for the advice, I hadn't seen the android folder in libsub.